src/foam/u2/view/UnstyledTableView.js

```diff
@@ -261,6 +262,12 @@ foam.CLASS({
         return this.__context__.columnConfigToPropertyConverter;
       }
     },
+    {
+      name: 'subStack',
+      factory: function() {
+        return foam.nanos.approval.NoBackStack.create({delegate: this.stack});
```

Comment:
Why do we do this?

Reply:
because we want real stack here not `NoBackStack` which table exports

Reply:
so on 'Back' we did really go back rather do nothing